tests/playbooks/tests_eth_pci_address_match.yml

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: pseudo-pci-test-only
+            persistent_state: present
+            type: ethernet
+            # `pci-0001:00:00.0` is a pseudo PCI address,
+            # which used only for testing purpose
+            match:
+              path:
+                - pci-0001:00:00.0
+            ip:
+              dhcp4: no
+              address:
+                - 192.0.2.3/24
+    - name: Verify nmcli connection profile
+      shell: |
+        set -euxo pipefail
+        nmcli -f match.path connection show pseudo-pci-test-only
+      register: match_path
+      ignore_errors: yes
+      changed_when: false
+
+    - name: "Assert that matching profile with interface at PCI address works"
+      assert:
+        that:
+          - "'pci-0001:00:00.0' in match_path.stdout"
+        msg: "Matching profile with interface at PCI address is failed"
```

Comment:
Please also add a test that modifies the match setting to remove the PCI address again and also add an always block that ensures that the test profile will be removed again. Please compare https://github.com/linux-system-roles/network/blob/main/tests/playbooks/tests_bond_deprecated.yml#L77

Reply:
If updated the connection profile and removed the match setting, then by default, the `name` will become `interface_name`, the test will raise error like "No such interface found".

Reply:
Updated.Thanks.